packages/babel-plugin-emotion/src/index.js

```diff
@@ -22,6 +31,30 @@ let primitivesStyledMacro = createStyledMacro({
   isWeb: false
 })
 
+let transformersSource = {
+  emotion: vanillaTransformers,
+  '@emotion/css': {
+    default: coreCssTransformer
+  },
+  '@emotion/core': {
+    // jsx isn't included here because it's handled differently
+    css: coreCssTransformer
+    // TODO: maybe write transformers for keyframes and Global
```

Comment:
what could be handled but is not?

Reply:
Minifying styles for keyframes and Global

Reply:
K, seems interesting to me so I'm going to work on those when this PR lands.